cicecore/drivers/nuopc/cmeps/ice_import_export.F90

```diff
@@ -177,22 +179,25 @@ subroutine ice_advertise_fields(gcomp, importState, exportState, flds_scalar_nam
     call fldlist_add(fldsToIce_num, fldsToIce, 'So_s'    )
     call fldlist_add(fldsToIce_num, fldsToIce, 'So_u'    )
     call fldlist_add(fldsToIce_num, fldsToIce, 'So_v'    )
-    call fldlist_add(fldsToIce_num, fldsToIce, 'Fioo_q'  )
+!    call fldlist_add(fldsToIce_num, fldsToIce, 'Fioo_q'  )
+    call fldlist_add(fldsToIce_num, fldsToIce, 'So_hmix'    )
+!    call fldlist_add(fldsToIce_num, fldsToIce, 'So_hmix' )
     if (flds_wiso) then
        call fldlist_add(fldsToIce_num, fldsToIce, 'So_roce_wiso', ungridded_lbound=1, ungridded_ubound=3)
     end if
 
     ! from atmosphere
     call fldlist_add(fldsToIce_num, fldsToIce, 'Sa_z'       )
-    call fldlist_add(fldsToIce_num, fldsToIce, 'Sa_u'       )
-    call fldlist_add(fldsToIce_num, fldsToIce, 'Sa_v'       )
-    call fldlist_add(fldsToIce_num, fldsToIce, 'Sa_shum'    )
-    call fldlist_add(fldsToIce_num, fldsToIce, 'Sa_tbot'    )
-    call fldlist_add(fldsToIce_num, fldsToIce, 'Sa_pbot'    )
+    call fldlist_add(fldsToIce_num, fldsToIce, 'Sa_u10m'    )
```

Comment:
@SmithJos13 We could not change the variable names. Those are also used by other configurations under UFS WM and CESM. This change will break those. Sometimes, Sa_u used like Sa_u10m sometimes not. So, please provide your data as Sa_u. So, you could do it in the CDEPS ERA5 mode and its steam file. We need to change only the required code and always thinking about other configurations and applications.